generator.yaml

```diff
@@ -131,6 +131,22 @@ resources:
         from:
           operation: RestoreDBClusterFromSnapshot
           path: SnapshotIdentifier
+      SourceDBClusterIdentifier:
+        from:
+          operation: RestoreDBClusterToPointInTime
+          path: SourceDBClusterIdentifier
+      RestoreType:
+        from:
+          operation: RestoreDBClusterToPointInTime
+          path: RestoreType
+      RestoreToTime:
+        from:
+          operation: RestoreDBClusterToPointInTime
+          path: RestoreToTime
+      UseLatestRestorableTime:
+        from:
+          operation: RestoreDBClusterToPointInTime
+          path: UseLatestRestorableTime
```

Comment:
I had a quick thought about making these something like:
```
      RestoreFrom:
        from:
          operation: RestoreDBClusterToPointInTime
          path: .
```
But it's just too verbose IMO..